apps/examples/webserver/webserver_main.c

```diff
@@ -99,12 +102,30 @@ static const char g_httpcnlost[] = "close";
 struct http_server_t *http_server = NULL;
 struct http_server_t *https_server = NULL;
 
+static void http_send_chunk(struct http_client_t *client)
+{
+	char msg_body[MAX_DATA_SIZE + 1] = {0, };
+
+	//add data
+	memset(msg_body, 0, MAX_DATA_SIZE + 1 );
+	memset(msg_body, 'a', MAX_DATA_SIZE);
+
+	http_send_response_chunk(client, 200, "OK", msg_body, NULL, FIRST_DATA);
+
+	for(int i = 0; i < MAX_DATA_COUNT; i++) {
```

Comment:
As per  https://wiki.tizen.org/Native_Platform_Coding_Idiom_and_Style_Guide#C_Coding_Style

> [M03] Keywords have following space rules. 
> • Put a space after (most) keywords
> o example: if, switch, case, for, do, while
> • Do not put a space after function like keywords
> o example: sizeof, typeof, alignof, __attribute__

Reply:
Because this is in the external, we don't need to follow TizenRT coding rules. But need to keep the same policy in the file.